### ISSUE TITLE
[FR] Please provide the feature to disable default mappings

### ISSUE THREAD

**Issue by yutkat** [renerocksai/telekasten.nvim#262]
**Please confirm**
- [x] I am running the latest version of this plugin

**Is your feature request related to a problem? Please describe.**

I use `<C-n>` to move to the next line in telescope, but in `Telekasten find_notes` it is overridden and doesn't work properly.

https://github.com/renerocksai/telekasten.nvim/blob/36dad2692974a5cc69813441442ccce8c53fbfd4/lua/telekasten.lua#L1656-L1663

**Describe the solution you'd like**

I would like to be able to disable the default mapping. 

**Describe alternatives you've considered**

It is relatively easy to override `FindNotes` by making `find_files_sorted` modular or non-local.

https://github.com/renerocksai/telekasten.nvim/blob/36dad2692974a5cc69813441442ccce8c53fbfd4/lua/telekasten.lua#L826

**Comment by lambtho12** [comment]
Indeed, we could provide an option for action_mappings. Not setting it would result in the current default, setting it to nil explicitly would disable them and providing a list of other mappings would overwrite the current settings.

This is a rather straight forward change, but I think I would rather implement it in the [refactoring branch](https://github.com/renerocksai/telekasten.nvim/tree/refact) so I do not need to do the work twice.
